### ISSUE TITLE
Can you provide the blender script used to render the images and depth maps used to train the SparseNeuS?

### ISSUE THREAD

**Issue by kcyt** [One-2-3-45/One-2-3-45#24]
Thanks for reading!

**Comment by Colin97** [comment]
@Dustinpro  Could you take a look?

**Comment by Dustinpro** [comment]
We have released our blender scripts for evaluation [here](https://github.com/One-2-3-45/One-2-3-45/tree/master/render).

**Comment by kcyt** [comment]
Thank you but how do we generate the groundtruth depth maps e.g. ("view_3_2_10_gt_depth_mm.png") used to train SparseNeuS? 

I assumed that if I want to test One-2-3-45's performance on a different dataset (e.g. 3D humans), I would have to train/finetune both the pre-trained Zero123 and the SparseNeuS. Do let me know if I am wrong.

**Comment by Dustinpro** [comment]
You can generate the depth map by uncommenting [this line](https://github.com/One-2-3-45/One-2-3-45/blob/ee278d0964ba2d3bf66f94d05da0f83a747ba6c1/render/single_render_eval.py#L204). 

However, in our camera-ready version, we used the mask loss instead of the depth loss. Thus, depth maps are not necessary for the training.

Yes, fine-tuning will improve the performance, especially for the multiview prediction module (Zero123), which may not work well for images of humans or faces.
